jenkins/deploy-clio-server.sh

```diff
@@ -141,9 +148,9 @@ stopcontainer
 # move /app.new to /app
 # create pos log directory if it doesn't already exist
 ${SSHCMD} <<-EOF
-  ([ ! -d ${APP_DIR}.old ] || sudo mv ${APP_DIR}.old ${APP_DIR}.old.old) &&
-  ([ ! -d ${APP_DIR} ] || sudo mv ${APP_DIR} ${APP_DIR}.old) &&
-  sudo mv ${APP_DIR}.new ${APP_DIR} &&
+  ([ ! -d ${PREVIOUS_APP_DIR} ] || sudo mv ${PREVIOUS_APP_DIR} ${PREVIOUS_APP_BACKUP_DIR}) &&
```

Comment:
Why do we need to `sudo` inside `ssh`? It seems like we should `ssh` to a user that already has the privs that we need (either by adding privs to an existing user or ssh as a different one). Since we can apparently `sudo` in our `ssh` session without entering a password there are no security issues with this as far as I can see.